src/pykeen/models/unimodal/node_piece.py

```diff
@@ -60,10 +104,21 @@ def __init__(
         embedding_specification = embedding_specification or EmbeddingSpecification(
             shape=(embedding_dim,),
         )
+        node_piece_kwargs = node_piece_kwargs or {}
+        node_piece_kwargs["k"] = num_tokens
+        if aggregation == "mlp":
+            # needs to be assigned to attribute to make sure that the trainable parameters are part of the model
+            # parameters
+            self.mlp = ConcatMLP(
+                num_tokens=num_tokens,
+                embedding_dim=embedding_specification.embedding_dim,
+            )
+            aggregation = self.mlp.forward
```

Comment:
I think just doing `self.mlp` is good since this is a callable and this is the preferred way to use the forward()

Reply:
done in afc873b4